server/api/hot_status.go

```diff
@@ -141,3 +143,30 @@ func (h *hotStatusHandler) GetHotStores(w http.ResponseWriter, r *http.Request)
 	}
 	h.rd.JSON(w, http.StatusOK, stats)
 }
+
+// @Tags hotspot
+// @Summary List the history hot regions.
+// @Accept json
+// @Produce json
+// @Success 200 {object} statistics.HistoryHotRegions
+// @Router /hotspot/regions/history [post]
```

Comment:
`get` should be more suitable.

Reply:
Too many parameters are needed, it is troublesome to switch to Get form

Reply:
I think GET can have body too.